.eslintrc.js

```diff
@@ -30,6 +30,13 @@
  * GitHub history for details.
  */
 
+const SIMPLIFIED_OOS_HEADER = `
```

Comment:
Do we need to call it SIMPLIFIED, how about if we just say OSS_HEADER, OSD_HEADER and remove `SIMPLIFIED`

Reply:
Agreed, we should make sure it's clear what the headers are for new files that OpenSearch Contributors have created versus the files that were brought over in the fork.

Reply:
Fair point. Ive updated the names to OSD_HEADER for this license string and changed the one below to be just OSS_HEADER, since thats used for files with an existing OSS header. Ive also added comments to make it easer to distinguish between the two in future.